### ISSUE TITLE
-H doesn't support multiple headers

### ISSUE THREAD

**Issue by gauravnarwani97** [hahwul/dalfox#253]
## Describe the bug
Hi Hahwul,
Multiple headers using -H not supported. while giving -H 'asd: asd' -H 'qwe: qwe' as input, it takes the last value i.e. qwe:qwe. I want to add multiple headers to the request. Could you look into this and let me know if I'm doing something wring?

dalfox url --deep-domxss --timeout 5 --silence -o asdasd.txt -H "X-asds: asdd" -H "X-asdqwe: qwe" --proxy http://127.0.0.1:8080 'http://testphp.vulnweb.com/listproducts.php?cat=1'

This adds only header- X-Asdqwe: qwe

## Environment
 * Dalfox Version: latest
 * Installed from: (e.g go-get/snapcraft/homebrew) go

**Comment by hahwul** [comment]
Hi @gauravnarwani97 
Thank you for submit issue!

Multiple headers are not currently supported. I'm aware of this issue, so I'll work on it.
Let’s roll up my sleeves, work this :D

**Comment by hahwul** [comment]
## Memo
https://github.com/hahwul/dalfox/blob/a12d1f90651f9b324ef63a995ff876ac8b5f8a87/pkg/optimization/optimization.go#L27

**Comment by hahwul** [comment]
## Test 
```
$ ./dalfox url https://xss-game.appspot.com/level1/frame\?q\=1 --proxy http://localhost:8090 -H "Test: 1234" -H "CCC: 11234" 
```

![111](https://user-images.githubusercontent.com/13212227/126196373-ea290b9e-6025-415e-b867-91c4cf32934c.jpg)

**Comment by hahwul** [comment]
Hello again @gauravnarwani97  
It was easier than I thought, so I solved it quickly! and I just released version `v2.4.6`. I'll close the this issue, so try it and if it's weird, please re-open it and tell me! Thank you very much for reminding me of the missing old issue :D

**Comment by gauravnarwani97** [comment]
Hey @hahwul ,
Wow that was super fast. The fix works fine. Good Job.
